bin/install

```diff
@@ -128,7 +128,7 @@ get_checksum_filename() {
 
 get_gpg_filename() {
   local -r version="$1"
-  echo "${toolname}_${version}_SHA256SUMS.sig"
+  echo "${toolname}_${version}_SHA256SUMS.72D7468F.sig"
```

Comment:
Yes, although it's not as easy as I'd like to do this in a cross-platform manner. My first shot at it was to do this:
 
```suggestion
  local -r keyid="$(gpg --dry-run --import --import-options show-only "${HASHICORP_KEY}" |
    head -n1 | awk '{ print $2 }' | tail -c 9)"
  echo "${toolname}_${version}_SHA256SUMS.${keyid}.sig"
```

but that only works on my Linux machine, and not my work Mac. On my Mac, GPG prints the key ID on the second line instead of the first line of output.

I suppose a more robust way would be to extract the information from the key itself. However, because this fixes a bug, I'd be tempted to ship this and dynamically extract the keyid later.

Reply:
Yeah, I agree, this is breaking a bunch of automation for me right now. Fully support fixing the issue hardcoding the current known key id, and doing separate work to pull the key id dynamically in the future.

Reply:
Should you need to workaround while we wait for the workflow to run (c'mon GitHub, it's been queued for an hour!), you can check out a specific plugin version like this:

```
$ git --git-dir ~/.asdf/plugins/terraform/.git checkout 18a14a379db86dbe640a5495a632bd28c1173418
```